lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -22,11 +22,24 @@ def self.run(params)
 
         if file_is_already_uploaded?(bucket, key)
           message = "File already exists in S3 bucket #{bucket} at #{key}"
-          if params[:skip_if_exists]
+
+          # skip_if_exists is deprecated but we want to keep backward compatibility.
+          if params[:if_exists].nil?
+            params[:if_exists] = if params[:skip_if_exists].nil? || params[:skip_if_exists] == false
+                                   :fail
+                                 else
+                                   :skip
+                                 end
```

Comment:
💄 Nitpick: testing the opposite condition (`== true`) would make it shorter because we won't have to test both `nil` and `false`

```suggestion
            params[:if_exists] = if params[:skip_if_exists] == true
                                   :skip
                                 else
                                   :fail
                                 end
```
Or even
```suggestion
            params[:if_exists] = (params[:skip_if_exists] == true) ? :skip : :fail
```

Just a matter of taste though, so up to you.

Reply:
17c6d6dde135966e00a7be1374e7736e4f657bb6